iotedgehubdev/certutils.py

```diff
@@ -498,3 +484,85 @@ def _get_kwargs_string(self, kwarg_key, min_length, max_length, default_str=None
                                                          min_length, max_length)
                 raise EdgeValueError(msg)
         return result_str
+
+    def _dump_cert_content(self, id_str, output_path, overwrite_existing=True):
```

Comment:
instead of passing id_str, how about passing cert_dict? Since here we already start to dump certs.